ang/civicase/activity/calendar/directives/activities-calendar.directive.js

```diff
@@ -90,7 +90,7 @@
        * container. Also unbinds the mouseup event in order to reduce the amount
        * of active DOM event listeners.
        *
-       * @param {Event} event DOM event triggered by the user mouse up action.
+       * @param {object} event DOM event triggered by the user mouse up action.
```

Comment:
for DOM events you can define them as this:

`* @param {document#event:mouseup} event DOM event triggered by the user mouse up action.`